sigs.yaml

```diff
@@ -3001,12 +3001,12 @@ workinggroups:
     - github: cantbewong
       name: Steve Wong
       company: VMware
-    - github: cindyxing
-      name: Cindy Xing
-      company: Microsoft
     - github: dejanb
       name: Dejan Bosanac
       company: Red Hat
+    - github: kate-goldenring
+      name: Kate Goldenring
+      company: Microsoft
     emeritus_leads:
```

Comment:
We should add Cindy to emeritus section

Reply:
yes I agree that Cindy deserves to go on the emeritus list unless she has some objection